src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -1422,8 +1422,28 @@
      * stylesheetfile</a>.
      */
     @Parameter( property = "stylesheetfile" )
-   private String stylesheetfile;
-
+    private String stylesheetfile;
+    
+    /**
+     * Specifies the path of an additional HTML stylesheet file.
+     * @since 3.1.2
+     */
+    @Parameter
+    private String addStylesheet;

```

Comment:
additionalStylesheet

Reply:
the name of the parameter is `--add-stylesheet`, the parameter should reflect that.